unittests/tools/test_risk_recon_parser.py

```diff
@@ -9,13 +11,13 @@ class TestRiskReconAPIParser(DojoTestCase):
 
     def test_api_with_bad_url(self):
         with open(get_unit_tests_scans_path("risk_recon") / "bad_url.json", encoding="utf-8") as testfile, \
-          self.assertRaises(Exception):
+          self.assertRaises(requests.exceptions.ConnectionError):
             parser = RiskReconParser()
             parser.get_findings(testfile, Test())
 
     def test_api_with_bad_key(self):
         with open(get_unit_tests_scans_path("risk_recon") / "bad_key.json", encoding="utf-8") as testfile, \
-          self.assertRaises(Exception):
+          self.assertRaises(Exception):  # noqa: B017 (Exception from tools/risk_recon/api.py --> def map_toes(self))
```

Comment:
In my opinion, this kind of `noqa` is not good. We should not tolerate "wrong implementation in code that we can change". It is tolerating the wrong implementation of another library. The issue is in the DD codebase.

I'm fine to add `noqa` with `TODO` note until [blind-except (BLE001)](https://docs.astral.sh/ruff/rules/blind-except/#blind-except-ble001) is solved.

Reply:
Good point :-)